src/main/java/com/artipie/api/RepositoryRest.java

```diff
@@ -73,17 +87,20 @@ public void init(final RouterBuilder rbr) {
      */
     private void getRepo(final RoutingContext context) {
         final RepositoryName rname = new RepositoryName.FromRequest(context, this.layout);
-        final ValidRepositoryName validator = new ValidRepositoryName(rname);
-        if (!validator.isValid()) {
+        final RepositoryNameValidator rnamecheck = new RepositoryNameValidator(rname);
```

Comment:
Here, we can use interface `Validator` rather than a class.

Reply:
@dgarus thanks for valuable notes. Could you please to take a look at changes?

Reply:
@olenagerasimova @dgarus please review, I hava added validators